pyramid_closure/closure/depswriter.py

```diff
@@ -30,8 +30,8 @@
 import shlex
 import sys
 
-import source
-import treescan
+from . import source
+from . import treescan
```

Comment:
Personally I don't like relative import... but finally I don't know what we should do ...